src/main/java/application/controller/IndexController.java

```diff
@@ -9,9 +14,19 @@
 
 @WebServlet("/")
 public class IndexController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("application");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    private void addAdmins() {
+        User admin = new User("admin", "admin", "12345");
+        admin.setRoles(Set.of(Role.of("ADMIN")));
+        userService.create(admin);
+    }
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        addAdmins();
```

Comment:
we will try to add admin every time we reach index page?

Reply:
done